src/components/ebay-breadcrumbs/index.marko

```diff
@@ -6,7 +6,7 @@ $ const {
     a11yHeadingTag,
     a11yMenuButtonText,
     class: inputClass,
-    items: attrTagItems,
+    item: attrTagItems = [],
```

Comment:
This should just be `items`, since that's the language you used for all of the other pluralizations.
```suggestion
    item: items = [],
```

Reply:
Leaving this as is, we need to convert the attrTagItems to items which is a true array.